assets/app/views/_pod-template.html

```diff
@@ -33,7 +33,7 @@
             </a>
             <span ng-if="build.spec.source">
               from
-              <span class="fa fa-code" aria-hidden="true"></span>
+              <span class="fa fa-code"><span class="sr-only"> source </span></span>
```

Comment:
I suggest moving `sr-only` span outside the icon span and adding `aria-hidden` to the icon. I believe on some platforms screen readers try to say something for the icon (iOS VoiceOver?).